### ISSUE TITLE
`ModuleNotFoundError: No module named 'pandas'` when running `pyolite - plotly.ipynb`

### ISSUE THREAD

**Issue by fortunewalla** [jupyterlite/jupyterlite#294]
## Description

```python
ModuleNotFoundError: No module named 'pandas'

```
## Reproduce

1. Open https://jupyterlite.readthedocs.io/en/latest/_static/lab/index.html
2. Open `pyolite - plotly.pynb`
3. `Run` --> `Run All Cells`
4. In the codeblock

```python
import plotly.graph_objects as go
import pandas as pd

from js import fetch

URL = "https://raw.githubusercontent.com/plotly/datasets/master/2014_usa_states.csv"

res = await fetch(URL)
text = await res.text()

filename = 'data.csv'

with open(filename, 'w') as f:
    f.write(text)

df = pd.read_csv(filename)

fig = go.Figure(data=[go.Table(
    header=dict(values=list(df.columns),
                fill_color='paleturquoise',
                align='left'),
    cells=dict(values=[df.Rank, df.State, df.Postal, df.Population],
               fill_color='lavender',
               align='left'))
])

fig.show()
```
5. The following error is displayed

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-3-a0d61a0f6f07> in <module>
      1 import plotly.graph_objects as go
----> 2 import pandas as pd
      3 
      4 from js import fetch
      5 

ModuleNotFoundError: No module named 'pandas'

```

## Expected behavior

I think the code assumes that `pandas` is already present and hence it is not explicitly installed. New users might not know how to install it.

## Context

- Operating System and version: win10
- Browser and version: Firefox 91 beta and also Edge Chromium 93 beta
- JupyterLite version: Version 0.1.0-alpha.6

<details><summary>Browser Output</summary>
<pre>
Paste the output from your browser Javascript console here.
</pre>
</details>

**Comment by FlyMyPG** [comment]
Change:  Add `pandas` import.
`await micropip.install('plotly==5.1.0')`
To:
`await micropip.install(['plotly', 'pandas'])`

**Comment by bollwyvl** [comment]
Fixed by #274
